R/utilities-pk-analyses.R

```diff
@@ -27,3 +27,20 @@ calculatePKAnalyses <- function(results, simulation) {
 toPKParameter <- function(netPKParameters) {
   toObjectType(netPKParameters, PKParameter)
 }
+
+
+#' Saves the pK-analyses  to csv file
+#'
+#' @param pkAnalyses pK-Analyses to exporte (typically calculated using \code{calculatePKAnalyses} or imported from file)
+#' @param simulation Instance of a simulation used to calculate the pK-Analyses
+#' @param filename Full path of where the pK-Analyses will be saved.
+#'
+#' @export
+exportPKAnalysesToCSV <- function(pkAnalyses, simulation, filename) {

```

Comment:
I think this is very confusing, passing the simulation all the time...

How about we store the reference to the simulation object in the `SimulationRersults` class, and a reference to the `SimulationRersults` object in the `SimulationPKAnalyses` class? This way the user could always have access to the specific results that were used for calculating the PK-analysis, and don't have to care about passing the simulation around?

This has not to be done in this PR, could open a separate issue.

Reply:
Yes this is a great suggestion. It will make the API easier to use for the user
I'll do it in this PR

Reply:
done. Sim is now a reference in SimPK and SimRes. That way we do not need to pass it as an extra parameter